components/automate-cli/cmd/chef-automate/automateConstants.go

```diff
@@ -36,3 +36,23 @@ const AUTOMATE_HA_ES_CAR = "automate-ha-elasticsidecar"
 const ORIGIN_PATTERN = `[a-zA-z0-9]*`
 const PACKAGE_NAME_PATTERN = `-[a-zA-Z0-9]*-.*-\d+\.`
 const RELEASE_AND_VERSION_PATTERN = `.*-(\d+\.\d+\.*\d*)-(\d{14})-.*\.hart$`
+
+const (
+	FRONTEND_COMMANDS = `
+	sudo chef-automate config patch /tmp/%s;
```

Comment:
why do we have space in front of the commands ? will this not cause any issue ?

Reply:
it was as it is in original commit, I have just move to constants file, it seem working for now